docs/core-concepts/rid-catalog.md

```diff
@@ -0,0 +1,111 @@
+# Runtime IDentifier (RID) catalog
+
+## What are RIDs?
+RID is short for *Runtime IDentifier*.RIDs are used to identify target operating systems where an application or asset 
+(i.e. assembly) will run. They look like this: "ubuntu.14.04-x64", "win7-x64", "osx.10.11-x64". 
+For the packages with native dependencies, as we saw, it will designate on which platforms the package can be restored. 
```

Comment:
as we saw where? I can land here from search. Maybe say as described in the xyz topic
it will designate -> the RID will designate